processor/src/lib.rs

```diff
@@ -1,12 +1,12 @@
-use std::{marker::Send, io::Cursor, collections::HashMap};
+use std::{collections::HashMap, io::Cursor, marker::Send};
 
 use async_trait::async_trait;
 use thiserror::Error;
 
 use frost::{curve::Curve, FrostError};
 
-mod coin;
-use coin::{CoinError, Coin};
+pub mod coin;
```

Comment:
Shouldn't be pub. I did make comments about splitting coin out of processor once though? Offering a generic multi-coin backend for a variety of purposes? Feel free to comment there.

Reply:
I think I needed to make it pub to access the ethereum functionality within `coins/ethereum`?

Reply:
ethers -> ethereum-serai -> processor
! ethers -> ethereum-serai <-> processor